scripts/run_train_ner.py

```
@@ -0,0 +1,13 @@
+import sys
+import os
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+from src.train_ner.train import train_ner_model
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)